packages/clay-color-picker/src/Hue.tsx

```diff
@@ -20,66 +20,56 @@ type Props = {
 	value: number;
 };
 
-const useIsomorphicLayoutEffect =
-	typeof window === 'undefined' ? React.useEffect : React.useLayoutEffect;
-
 /**
  * Renders Hue component
  */
 const ClayColorPickerHue = ({value = 0, onChange = () => {}}: Props) => {
 	const containerRef = React.useRef<HTMLDivElement>(null);
-	const selectorActive = React.useRef<boolean>(false);
 
-	const {onPointerMove, setXY, x, y} = usePointerPosition(containerRef);
+	const {setXY, y} = usePointerPosition(containerRef);
 
-	const removeListeners = () => {
-		selectorActive.current = false;
+	const [internalValue, setInternalValue] = useState<number>(value);
 
-		window.removeEventListener('pointermove', onPointerMove);
-		window.removeEventListener('pointerup', removeListeners);
-	};
+	const handleOnChangeEnd = (event: any) => {
+		onChange(internalValue);
+
+		setXY({x: internalValue, y});
 
-	useIsomorphicLayoutEffect(() => {
-		if (containerRef.current && selectorActive.current) {
-			onChange(xToHue(x, containerRef.current));
-		}
-	}, [x]);
+		event.target.blur();
+
+		event.target.focus();
```

Comment:
I don't understand very well why we need this 😅, could you explain it better?

Reply:
I think there is a bug in FF input range with `onPointerUp`. It keeps dragging the slider thumb even when the left mouse button is released. This didn't happen when I used `onMouseUp`. The only way I could get it to work properly was blur then focus.

Reply:
Hmm this is weird, I'll do some testing.